README.md

```diff
@@ -155,11 +153,10 @@ To run a genome wide analysis on 1K genomes on a reasonable time, make sure you
 
 ```
 export SPARK_CLASSPATH=googlegenomics-spark-examples-assembly-1.0.jar 
-spark-class -Dspark.shuffle.spill=true \
-com.google.cloud.genomics.spark.examples.VariantsPcaDriver \
---spark-master spark://hadoop-m:7077 \
---jar-path googlegenomics-spark-examples-assembly-1.0.jar \
---client-secrets ./client_secrets.json \
+spark-submit --class com.google.cloud.genomics.spark.examples.VariantsPcaDriver \
+--master spark://hadoop-m:7077 --conf spark.shuffle.spill=true \
```

Comment:
`--master` is still needed as that's the spark-submit parameter.

I'm leaving `--spark-master` available so it can be used from the sbt command line.